misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractIterableToContainInAnyOrderOnlyValuesExpectationsTest.kt

```diff
@@ -0,0 +1,375 @@
+package ch.tutteli.atrium.specs.integration
+
+import ch.tutteli.atrium.api.fluent.en_GB.exactly
+import ch.tutteli.atrium.api.fluent.en_GB.message
+import ch.tutteli.atrium.api.fluent.en_GB.notToContain
+import ch.tutteli.atrium.api.fluent.en_GB.toContain
+import ch.tutteli.atrium.api.fluent.en_GB.toThrow
+import ch.tutteli.atrium.api.fluent.en_GB.value
+import ch.tutteli.atrium.api.fluent.en_GB.values
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.logic.creating.iterablelike.contains.reporting.InAnyOrderOnlyReportingOptions
+import ch.tutteli.atrium.specs.*
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.additionalElements
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.anElementWhichEquals
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.emptyInAnyOrderOnlyReportOptions
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.fluentEmpty
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.mismatches
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.mismatchesAdditionalElements
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.oneToEleven
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.oneToFour
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.sizeDescr
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.toContainInAnyOrderOnly
+import ch.tutteli.atrium.specs.integration.IterableToContainSpecBase.Companion.toContainSize
+import ch.tutteli.atrium.specs.integration.utils.SubjectLessTestData
+import ch.tutteli.atrium.testfactories.TestFactory
+
+@Suppress("FunctionName")
+abstract class AbstractIterableToContainInAnyOrderOnlyValuesExpectationsTest(
+    private val toContainInAnyOrderOnlyValuesSpec: Fun3<Iterable<Double>, Double, Array<out Double>, InAnyOrderOnlyReportingOptions.() -> Unit>,
+    private val toContainInAnyOrderOnlyNullableValuesSpec: Fun3<Iterable<Double?>, Double?, Array<out Double?>, InAnyOrderOnlyReportingOptions.() -> Unit>
+) : ExpectationFunctionBaseTest() {
+
+    @TestFactory
+    fun subjectLessTest() = subjectLessTestFactory(
+        SubjectLessTestData(
+            toContainInAnyOrderOnlyValuesSpec.forSubjectLessTest(
+                2.5,
+                arrayOf(),
+                emptyInAnyOrderOnlyReportOptions)
```

Comment:
can you please hit CTLR+ALT+L to format the file by intellij

Reply:
@robstoll - Done.